Removed unnecessary codes and modify a code lines to support to UNIT coin.

jeong760 opened the pull request:
Removed unused Preinfo.cs class code from on /src/CoiniumServ/Daemon/Responses

Modify or add a code to support a UNIT(Universal Coin)

Add a new PropertyConvertor.cs class on /srv/CoiniumServ/Daemon/Converters/PropertyConverter.cs

Add a following code on MininInfo.cs on /srv/CoiniumServ/Daemon/Responses

[JsonProperty("netmh/s")] // added to support unit and possibly some other coins
private double NetMHps { get; set; }
Modify a following code on src\CoiniumServ\Daemon\DaemonBase.cs

using CoiniumServ.Daemon.Converters;
// process response with converter. needed for all coin wallets which gives non-standard info.
string jsonLC = PropertyConverter.DeserializeWithLowerCasePropertyNames(json).ToString();
_logger.Verbose("rx: {0}", jsonLC.PrettifyJson());
try
{
return JsonConvert.DeserializeObject<DaemonResponse>(jsonLC);
}

Specially thanks to @flyx2888 to modify and add a new class for UNIT coin.

Regards,
John Ahn

jeong760 committed:
The following code has been changed.

- Removed unnecessary PreInfo.cs code
- Added a new PropertyConvertor.cs
- Modify the following codes to support a UNIT coin
  - MiningInfo.cs
  - DaemonBase.cs

 Specially thanks to @flyx2888 for UNIT coin code modification.

jeong760 committed:
Removed unnecessary code related wtih PreInfo.cs

jeong760 commented:
@bonesoul  I've opened a new PR for previous change. please apply this PR

bonesoul commented:
okay lot better.

thanks

bonesoul commented:
though the tests are failing;

https://circleci.com/gh/bonesoul/CoiniumServ/281?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

`CSC : error CS2001: Source file '/temp/src/CoiniumServ/Daemon/Converters/PropertyConverter.cs' could not be found. [/temp/src/CoiniumServ/CoiniumServ.csproj]`

i guess u forgot to include the file?

jeong760 committed:
Fixed missing a PropertyConverter.cs

jeong760 commented:
@bonesoul  Sorry. it was remaining on untracked file status.  I forgot to issue a **git add .**

jeong760 commented:
@bonesoul 

Just my private opinion but I want to call this of CoiniumServ as ver 2.6.4 to fixed various issues.
Because after you released a version 2.6.3 we've fixed various issues on it.

jeong760 committed:
Update footer.cshtml

Fixed wrong URL information about CoiniumServ on /srv/web/default/views/layout/footer.cshtml

jeong760 committed:
Update donate.cshtml

Removed Coinbase.com button it seems does not work either so. 

<a class="coinbase-button" data-code="cf6bb9334d09f07ad3abbe5894702ae9" data-button-style="custom_small" href="https://coinbase.com/checkouts/cf6bb9334d09f07ad3abbe5894702ae9">Donate Bitcoins</a>
                <script src="https://coinbase.com/assets/button.js" type="text/javascript"></script>

jeong760 committed:
Update donate.cshtml

jeong760 committed:
Update donate.cshtml

<li>Litecoin: ---> <li>Ethereum:

jeong760 committed:
Update donate.cshtml

<a href="http://www.coiniumserv.com" target="_blank">  ---> https://github.com/bonesoul/CoiniumServ

flyx2888 commented on src/web/default/views/donate/donate.cshtml:
```diff
@@ -30,7 +30,7 @@
                 <ul>
                     <li>Bitcoin: <a href="bitcoin:1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1?label=CoiniumServ%20Donation"><code>1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1</code></a></li>
-                    <li>Litecoin: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
+                    <li>Ethereum: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
```
@jeong760 

This is still wrong. The eth address in the href does not match the displayed address. It's set to the litecoin address.

jeong760 replied:
@flyx2888 Yes but donation page displayed an Ethereum address. so this, not the issue.
![capture](https://user-images.githubusercontent.com/18043565/39081147-1fe2b2f4-4577-11e8-9625-541fc6b10944.JPG)

flyx2888 replied:
@jeong760 

It's not an issue if they manually copy and paste. It's an issue if they click on the link.

jeong760 committed:
Update donate.cshtml

Fixed Ethereum address in the code.

0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b

jeong760 committed:
Update donate.cshtml

0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b

jeong760 committed:
Update stronghands-example.json

coin": "stronghands.json"

jeong760 committed:
Update stronghands.json

jeong760 commented:
@bonesoul,

Do you any reasons to hesitate to merged this PR?

Regards,
John Ahn

bonesoul commented:
@jeong760 nope i totally forgot about it, lemme check it and merge then if it's all okay.
